pkg/local_object_storage/metabase/inhume.go

```diff
@@ -73,6 +73,11 @@ var errBreakBucketForEach = errors.New("bucket ForEach break")
 // if at least one object is locked.
 func (db *DB) Inhume(prm *InhumePrm) (res *InhumeRes, err error) {
 	err = db.boltDB.Update(func(tx *bbolt.Tx) error {
+		garbageBKT, err := tx.CreateBucketIfNotExists(garbageBucketName)
```

Comment:
Do this in `Init`?

Reply:
if fact it is already [called](https://github.com/nspcc-dev/neofs-node/pull/1318/files#diff-1ff37983e3b5b6c80d5873c9922f36816780fb8cfbd693b52aa539543864da52R57); changed to just `Bucket`